graphql/mutations/auth/RegisterUserMutation.gql

```diff
@@ -4,13 +4,15 @@ mutation registerUserMutation(
   $email: String!
   $password: String!
   $confirmedPassword: String!
+  $turnstileToken: String!
```

Comment:
Ah yes - see [this comment](https://github.com/BristolSTA/uobtheatre-api/pull/820#discussion_r1789735189) on the API PR - but definitely then we should tweak that on the API to match.